src/convo_politeness.py

```diff
@@ -202,7 +202,7 @@ def train_polite(comments):
 # input: a pandas dataframe: _id, text
 # output: a pandas dataframe: _id, text, politeness
 def get_politeness_score(comments):
-  logging.info("Beging calculating politeness, rows: {}".format(len(comments)))
+  logging.info("Begining calculating politeness, rows: {}".format(len(comments)))
```

Comment:
nit: Beginning, an example of Muphry's Law

Reply:
fixed :p